.github/workflows/check-unity-sdk-for-updates.yml

```
@@ -0,0 +1,58 @@
+          git add .
+          git commit -m 'build: update adgem sdk dependency version'
+          git push origin HEAD:refs/heads/update-adgem-sdk-$LATEST_ANDROID_SDK_VERSION
+
+      - name: Create Pull Request
+        if: env.HAS_CHANGES
+        uses: peter-evans/create-pull-request@v7
+        with:
+          token: ${{ secrets.PUBLIC_REPO_BOT_GITHUB_TOKEN }}
+          branch: update-adgem-sdk-${{ env.LATEST_ANDROID_SDK_VERSION }}
+          title: 'build: update adgem sdk dependency version'
+          body: | 
```

yamllint

[error] trailing spaces

(trailing-spaces)